src/Controller/Admin/ModuleRecommendedController.php

```diff
@@ -93,4 +101,27 @@ public function indexAction(): JsonResponse
 
         return $response;
     }
+
+    /**
+     * @param string $tabClassName
+     *
+     * @return string|null
+     */
+    protected function getAddonsLink(string $tabClassName): ?string
+    {
+        $utm = '?utm_source=back-office&utm_medium=dispatch&utm_content=download&utm_campaign=back-office-' . $this->context->getLanguage()->iso_code;
+
+        switch ($tabClassName) {
+            case 'AdminCarriers':
+                $link = $this->trans('https://addons.prestashop.com/en/518-shipping-logistics', 'Modules.Mbo.Modulescatalog');
```

Comment:
there an `/en/` here to replace with the iso_code

Reply:
In french, the page is `/fr/518-livraison-logistique`
In Italian : `/it/518-spedizioni-logistica`
And so on...
That's why I put the link in the translation module.
Each language may update the URL of those pages.